app/views/user_email_mailer/results_email.html.erb

```diff
@@ -1,74 +1,69 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Email letter</title>
-  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
-  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
-  <style>
-      * {
-          margin: 0;
-          padding: 0;
-          box-sizing: border-box;
-          font-family: 'Josefin Sans', 'Arial', 'Ubuntu',  sans-serif;
-          font-style: normal;
-          font-weight:700;
-          text-align: center;
-          line-height: 0;
-      }
-      .board{
-          min-width:1080px;
-      }
-      .lnk-unsubscribe{
-          text-align: left;
-      }
-  </style>
-  <%= stylesheet_link_tag 'email_page' , media: 'all' %>
+	<title>Email letter</title>
+	<meta content='text/html; charset=UTF-8' http-equiv='Content-Type'/>
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
I'm pretty sure most email programs will not permit this. From what I remember, all CSS has to be included in the email body without any external fetches.

Reply:
I removed the CDN links from the code and updated the styles to improve mobile responsiveness